unit-tests/src/main/scala/java/lang/IntegerSuite.scala

```diff
@@ -0,0 +1,38 @@
+package java.lang
+
+object IntegerSuite extends tests.Suite {
```

Comment:
I'm missing some test for the corner cases: `0`, `Integer.MAX_VALUE` and `Integer.MIN_VALUE`